skyportal/models/tag.py

```diff
@@ -0,0 +1,33 @@
+import sqlalchemy as sa
+from sqlalchemy.orm import relationship
+
+from baselayer.app.models import AccessibleIfUserMatches, Base, join_model
+from skyportal.models import User
+from skyportal.models.obj import Obj
+
+
+class ObjTagOption(Base):
+    """A record of a list of tags that are available."""
+
+    name = sa.Column(sa.String, nullable=False, doc="Tags list.", unique=True)
+
+
+ObjTag = join_model("obj_tag", ObjTagOption, Obj)
+
+read = update = delete = AccessibleIfUserMatches("user")
```

Comment:
this doesn't do anything just written in the middle of a file. It's supposed to be fields of a python class you're applying this to. Right now it's just creating unused variables. Let's just remove it and worry about data access later.